website/docs/components/form/file-input/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+A file input enables the user to select a file from their computer.
```

Comment:
Isn't this redundant, looking at the whole page/content? It's kinda a repetition of what I see in the page subtitle, as well as the "When to use" below